### ISSUE TITLE
this error is coming in google colab:

### ISSUE THREAD

**Issue by 24thTinyGiant** [bcmi/DCI-VTON-Virtual-Try-On#36]
/content/DCI-VTON-Virtual-Try-On
2023-12-05 14:49:45.898989: E tensorflow/compiler/xla/stream_executor/cuda/cuda_dnn.cc:9342] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2023-12-05 14:49:45.899048: E tensorflow/compiler/xla/stream_executor/cuda/cuda_fft.cc:609] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2023-12-05 14:49:45.899086: E tensorflow/compiler/xla/stream_executor/cuda/cuda_blas.cc:1518] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2023-12-05 14:49:47.701705: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Global seed set to 23
Loading model from /content/DCI-VTON-Virtual-Try-On/viton512.ckpt
Global Step: 58240
LatentTryOnDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.54 M params.
making attention of type 'vanilla' with 512 in_channels
Working with z of shape (1, 4, 64, 64) = 16384 dimensions.
making attention of type 'vanilla' with 512 in_channels
Downloading: "https://github.com/DagnyT/hardnet/raw/master/pretrained/train_liberty_with_aug/checkpoint_liberty_with_aug.pth" to /root/.cache/torch/hub/checkpoints/checkpoint_liberty_with_aug.pth
100% 5.10M/5.10M [00:00<00:00, 64.6MB/s]
config.json: 100% 4.52k/4.52k [00:00<00:00, 18.9MB/s]
model.safetensors: 100% 1.71G/1.71G [00:24<00:00, 68.9MB/s]
^C

why is this error coming in google colab I'm using V100 gpu .

**Comment by PraNavKumAr01** [comment]
Hey I'm running into the same issue, have you found a solution yet?

**Comment by PraNavKumAr01** [comment]
@Limbor any solution? I've even tried the testing on just 1 image, still runs into a OOM
